test/tomo/backends/astra_cpu_test.py

```diff
@@ -31,9 +31,7 @@
 from odl.tomo.util.testutils import skip_if_no_astra
 
 
-# TODO: test scaling of ASTRA projectors
```

Comment:
I moved the scaling tests to the Xray trafo tests in order to have them all (cpu + cuda) in one place. Nevertheless the tests have to be improved.